Tests/LibC/TestPThreadPriority.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2022, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+#include <pthread.h>
+#include <sched.h>
+#include <unistd.h>
+
+#define TEST_CASE_IN_PTHREAD(x)                                                 \
+    static void* __TESTCASE_FUNC(x##__inner)(void*);                            \
+    TEST_CASE(x)                                                                \
+    {                                                                           \
+        pthread_t thread;                                                       \
+        pthread_create(&thread, nullptr, __TESTCASE_FUNC(x##__inner), nullptr); \
+        pthread_join(thread, nullptr);                                          \
+    }                                                                           \
+    static void* __TESTCASE_FUNC(x##__inner)(void*)
+
+TEST_CASE_IN_PTHREAD(basic_priority)
+{
+    auto min_priority = sched_get_priority_max(0);
```

Comment:
This should probably be `sched_get_priority_min`.

Reply:
:facepalm: